sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodParameterUnused.AfterCSharp8.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Tests.TestCases
```

Comment:
I think we could add a test with switch expression and tuple deconstruction to ensure C#8 syntax works correctly for theses syntaxes on this rule too

Reply:
I'll skip Tuple deconstruction after offline discussion.